src/kem/kyber/kem_kyber_512.c

```diff
@@ -28,9 +28,9 @@ OQS_KEM *OQS_KEM_kyber_512_new() {
 	return kem;
 }
 
-OQS_STATUS PQCLEAN_KYBER512_CLEAN_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
-OQS_STATUS PQCLEAN_KYBER512_CLEAN_crypto_kem_enc(unsigned char *ct, unsigned char *ss, const unsigned char *pk);
-OQS_STATUS PQCLEAN_KYBER512_CLEAN_crypto_kem_dec(unsigned char *ss, const unsigned char *ct, const unsigned char *sk);
+int PQCLEAN_KYBER512_CLEAN_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
```

Comment:
What is the reason for this change? Is it just that `copy_from_pqclean/src/kem/family/kem_scheme.c` was not updated?

Reply:
Yeah, seems to have been some previous inconsistency.  The `PQCLEAN_` functions from PQClean do actually return an int, so we should have that be their interface when we reference their prototype, and then cast to `OQS_STATUS` later when we actually call into them, which is I believe what this PR now accomplishes correctly.